account_invoice_start_end_dates/models/account_invoice.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2013-2016 Akretion, Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError, ValidationError
+
+
+class AccountInvoiceLine(models.Model):
+    _inherit = 'account.invoice.line'
+
+    start_date = fields.Date('Start Date')
```

Comment:
The attribute string is redundant. String parameter equal to name of variable (attribute-string-redundant)
```suggestion
    start_date = fields.Date()
```